src/vs/workbench/contrib/positronConsole/browser/positronConsoleActions.ts

```diff
@@ -1062,3 +1099,22 @@ export function registerPositronConsoleActions() {
 		}
 	});
 }
+
+
+/**
+ * Register the internal command for executing code in console from the extension API.
+ * This command is called by the positron.executeCodeInConsole API command.
+ */
+CommandsRegistry.registerCommand('_executeCodeInConsole', async (accessor, ...args: [string, URI, IPosition]) => {
```

Comment:
This command is registered here so we can make one in `extHostApiCommands.ts`. I want to note that right now these are _not_ optional arguments for using with this command. Good? Bad? We could do these arguments like `URI | undefined` if we want.

Reply:
I believe I'm happy with the arguments being required. The visual editor will always want to pass both URI and position.